ajna/v2/urls.py

```diff
@@ -172,4 +172,14 @@
         auctions.AuctionsActiveView.as_view(),
         name="auctions-active",
     ),
+    path(
+        "auction/<uid>/",
```

Comment:
url's are usually plural. Change these two to `auctions` and the other one to `auctions/<auction_uid>/takes/`